src/init.rs

```diff
@@ -0,0 +1,19 @@
+use anyhow::Result;
+use gettextrs::{setlocale,LocaleCategory};
+use log::info;
+use std::fs::File;
+use std::io::Write;
+
+/// Unset the locale for the current process.
+pub fn unset_locale() {
+    setlocale(LocaleCategory::LcAll, "");
+}
+
+/// Helper to adjust the OOM score of the currently running process.
+pub fn set_oom(score: &str) -> Result<()> {
+    // Attempt adjustment with best-effort.
+    if let Err(err)  = File::create("/proc/self/oom_score_adj")?.write_all(score.as_bytes()) {
```

Comment:
nit, If an async function calls low level primitives then we should always try the tokio::async primitive with an await. Thus a recommendation to change this function to be async. wdyt of the general practice? I worry about mixing async/non-async code, when we could always add async code to begin with.

Reply:
personally I'm more of a fan of doing it synchronously here, as we want to do this operation before proceeding with the program execution. from my perspecitve, the advantage of async would be lost because we'd immediately await anyway, thus basically making it synchronous (i think). I'm open to to discussing though

Reply:
That makes sense as well. Not a blocker to this PR.